src/elements/content-sidebar/SidebarSection.scss

```diff
@@ -27,12 +31,8 @@
             fill: $twos;
         }
     }
-
-    .icon-caret-down {
-        transform: rotateZ(-90deg);
-    }
 }
 
 .be .bcs-section-open .btn-plain.bcs-section-title .icon-caret-down {
-    transform: none;
+    transform: rotateZ(180deg);
```

Comment:
Could we pull this into a shared variable. Something like $bdl-caret-transform?

Reply:
I am not sure if we should variable-ize everything. No gaurantee that we will keep that degree forever.